src/public/js/index.js

```diff
@@ -0,0 +1,28 @@
+import getTimeRemaining from './countdown';
+
+const initializeClock = (minutes) => {
+  document.getElementById('start').style.display = 'none';
+  document.getElementById('stop').style.display = 'block';
```

Comment:
If we add and remove a "hidden" `class` (which adds the display `none` property. Then style will be inherited by whatever style it had before, therefore we are not restricted to keeping the element as a block.

This way however, we will overwrite our stylesheet and have to change the style by changing this bit of javascript.